hydradx/tests/test_omnipool_amm.py

```diff
@@ -2022,30 +2022,56 @@ def test_volatility_limit(omnipool: oamm.OmnipoolState):
 
 
 @given(omnipool_reasonable_config(), st.floats(min_value=0.01, max_value=0.1), st.floats(min_value=0.01, max_value=0.1))
-def test_withdraw_size_limit(omnipool: oamm.OmnipoolState, max_withdrawal_per_block, max_lp_per_block):
+def test_LP_limits(omnipool: oamm.OmnipoolState, max_withdrawal_per_block, max_lp_per_block):
```

Comment:
Eventually we should split this into 2 tests right? One for the withdrawal limit, one for the add limit? This is fine for now though.